src/main/java/org/skriptlang/skript/lang/script/ScriptWarning.java

```diff
@@ -57,9 +56,9 @@ public enum ScriptWarning {
 	}
 
 	ScriptWarning(String warningName, String pattern, @Nullable String deprecationMessage) {
+		this.deprecationMessage = deprecationMessage;
 		this.warningName = warningName;
 		this.pattern = pattern;
-		this.deprecationMessage = deprecationMessage;
```

Comment:
the order should follow the order of parameters (no reason to change the existing order)

Reply:
It matches the field order and is sorted pretty

Reply:
Huh? You just changed the field order? It matched before as well?

Reply:
Looks better

Reply:
Yes, there's no reason to change it, and other classes follow the parameters.
If you were writing a new class, whatever, but there's no reason to change it now.